src/Hangman.java

```diff
@@ -0,0 +1,82 @@
+import java.util.List;
+
+public class Hangman {
+
+    private List<Character> characters;
+    private List<Character> coveredWord;
+    private List<Character> copyCharacters;
```

Comment:
copyCharacters? What is that?

Reply:
I need third List where i copy character list because some words contain more than one letter of this same name, and when i use method set(index,character) it change only first in index and the next letter is still covered by '*'.  I used that List in class Hangman in method changeStarByCharacter().